bundles/org.openhab.binding.sonnen/src/main/java/org/openhab/binding/sonnen/internal/communication/SonnenJsonPowerMeterDataDTO.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sonnen.internal.communication;
+
+/**
+ * The {@link SonnenJsonPowerMeterDataDTO} is the Java class used to map the JSON
+ * response from the API to a PowerMeter Object.
+ *
+ * @author Christian Feininger - Initial contribution
+ */
+public class SonnenJsonPowerMeterDataDTO {
```

Comment:
Please either annotate with `@NonNullByDefault` or move this DTO class to a folder named "dto" to have it excluded from null annotation SAT checks.

Reply:
added